Readability.js

```diff
@@ -295,6 +295,9 @@ Readability.prototype = {
 
       // Standard relative URI; add entire path. pathBase already includes a
       // trailing "/".
+      if (baseElementPath)
+        return toAbsoluteURI(baseElementPath) + uri;
```

Comment:
This is recursive if the `baseElementPath` is something that doesn't hit the earlier return statements, which seems like it would lead to infinite recursion if passed the 'wrong' string.

Reply:
It may be worth having a test case for a relative plain `<base>` element as well (e.g. `<base href="foopath/"/>`).